redis/test_test.go

```diff
@@ -54,7 +54,61 @@ type Server struct {
 	done chan struct{}
 }
 
+type Version struct {
+	major int
+	minor int
+	micro int
+}
```

Comment:
As this not needed outside the package there's no need to export it so lower case the type name.

Also switching to semantic version naming of the parameters.
```suggestion
type version struct {
	major int
	minor int
	patch int
}
```